test/new-e2e/tests/updater/linux_test.go

```diff
@@ -100,6 +100,11 @@ func (v *vmUpdaterSuite) TestUpdaterDirs() {
 }
 
 func (v *vmUpdaterSuite) TestInstallerUnitLoaded() {
+	systemdPath := "/lib/systemd/system"
```

Comment:
is it useful to enforce the systemd path?
If so it needs to be enforced on write

Reply:
I added it because the e2e test for centos/redaht was failing with "Failed to get unit file state for datadog-installer.service: No such file or directory", so I added it check if the file really was there or not.
Turns out it's not there even though the cp command puts it there during the boostrap process (I checked by `ls` -ing it in posttrans.sh after the bootstrap), so it looks like rpm is somehow deleting the file after the installation?

Reply:
Since we are releasing first deb, the best is probably to skip temporarely centos/redhat and work on them just after